starfish/test/image/test_imagestack.py

```diff
@@ -158,6 +158,12 @@ def test_from_numpy_array_raises_error_when_incorrect_dims_passed():
         ImageStack.from_numpy_array(array.reshape((1, 1, 1, 1, 2, 2)))
 
 
+def test_from_numpy_array_automatically_handles_float_conversions():
+    x = np.zeros((1, 1, 1, 20, 20), dtype=np.uint16)
+    stack = ImageStack.from_numpy_array(x)
+    assert (stack.numpy_array.dtype == np.float32)
```

Comment:
remove parenthesis.

Reply:
Also, once you make tony's other change, verify that a warning is generated.